app/controllers/katello/api/v2/repositories_controller.rb

```diff
@@ -128,6 +133,14 @@ def create
       respond_for_show(:resource => repository)
     end
 
+    api :GET, "/repositories/repository_types", N_("Show the available repository types")
+    param :creatable, :bool, :desc => N_("When set to 'True' repository types that are creatable will be returned")
```

Comment:
Does `to_bool` handle capitalized True/False or should we be suggesting users use `'true' ?